Update WAS integration logos and Readme (#3257)
DataDog/integrations-core: ibm_was/README.md

## Overview

This check monitors [IBM Websphere Application Server (WAS)][1] through the Datadog Agent. This check supports IBM WAS versions >= 8.5.5.

## Setup

The IBM WAS Datadog integration collects enabled PMI Counters from the WebSphere Application Server environment. Setup requires enabling the PerfServlet, which provides a way for Datadog to retrieve performance data from WAS.

> By default, this check collects JDBC, JVM, thread pool and Servlet Session Manager metrics. You may optionally specify additional metrics to collect via the the "custom_queries" section, as demonstrated in the [example check configuration][2].

Reviewer: ```suggestion
By default, this check collects JDBC, JVM, thread pool, and Servlet Session Manager metrics. You may optionally specify additional metrics to collect in the "custom_queries" section. See the [sample check configuration][2] for examples.
```

### Installation

The IBM WAS check is included in the [Datadog Agent][3] package.

#### Enable the PerfServlet

> The servlet's .ear file (PerfServletApp.ear) is located in the WAS_HOME/installableApps directory, where WAS_HOME is the installation path for WebSphere Application Server.

Reviewer: ```suggestion
The servlet's .ear file (PerfServletApp.ear) is located in the `<WAS_HOME>/installableApps` directory, where `<WAS_HOME>` is the installation path for WebSphere Application Server.
```

The performance servlet is deployed exactly as any other servlet. Deploy the servlet on a single application server instance within the domain.

> Note: Starting with version 6.1, you must enable application security to get the PerfServlet working.

Reviewer: ```suggestion
**Note**: Starting with version 6.1, you must enable application security to get the PerfServlet working.
```

### Configuration

1. Edit the `ibm_was.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to collect your IBM WAS performance data. See the [sample ibm_was.d/conf.yaml][2] for all available configuration options.

2. [Restart the Agent][4].

#### Log Collection

Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:
```
    logs_enabled: true
```

> Next, point the config file to the proper WAS log files. You can uncomment the lines at the bottom of the WAS integration's config file, and amend them as you see fit:

Reviewer: ```suggestion
Next, edit `ibm_was.d/conf.yaml` by uncommenting the `logs` lines at the bottom. Update the logs `path` with the correct path to your WAS log files. 
```
